.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
+    steps:
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
```

Comment:
this is a debug line, do we want this?

Reply:
I'm also worried that this may contain secrets ...

Reply:
It does hide sensitive information, so that should not be an issue. It is a good question wether we want to do this or not. Maybe we can just leave it out for now and add it later if we find a use for it!